.ci_support/migrations/jpeg_to_libjpeg_turbo.yaml

```diff
@@ -0,0 +1,15 @@
+migrator_ts: 1678152141
+__migrator:
+  kind: version
+  migration_number: 3
+  bump_number: 1
+  commit_message: "Rebuild for jpegturbo migration"
+  paused: false
+  pr_limit: 1
+
+libjpeg_turbo:
+  - 2.1.5
```

Comment:
OK, so I read up a bit on the upgrade [path](https://github.com/conda-forge/conda-forge.github.io/issues/673#issuecomment-1426881229) plus current issues/PRs, and this has picked up the most recent changes from https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4199, so I think we're good here.